app/views/idv/address/new.html.erb

```diff
@@ -42,7 +42,7 @@
                             label: t('idv.form.zipcode'), required: true,
                             pattern: '(\d{5}([\-]\d{4})?)',
                             wrapper: false,
-                            input_html: { aria: { invalid: false }, class: 'usa-input', value: @pii['zipcode'] } %>
+                            input_html: { aria: { invalid: false }, value: @pii['zipcode'] } %>
```

Comment:
This styling was making the ZIP code input text smaller than the other inputs:
<img width="484" alt="Screen Shot 2022-07-04 at 3 51 02 PM" src="https://user-images.githubusercontent.com/9039672/177223252-9ca637a3-c3e2-4faf-a559-49917b90a429.png">